tests/test_admin.py

```diff
@@ -443,6 +440,14 @@ def setUp(self):
         self.versionable = PollsCMSConfig.versioning[0]
         self.version_admin = admin.site._registry[self.versionable.version_model_proxy]
 
+
+    def test_get_preview_url(self):
+        content = factories.PollContentWithVersionFactory(language="fr")
+        self.assertEqual(
```

Comment:
I feel like this needs an additional explicit check to verify French is the language of the preview.

Reply:
@marksweb I reverted this change, since it needs to go into the core (where tests have been adjusted). See https://github.com/django-cms/django-cms/pull/7691/files.